Could not find a declaration file for module 'slimdom'

I discovered this error while upgrading `slimdom-sax-parser`'s dependency from `2.3.2` to `3.0.0`.

My (TSDX-based) setup there began complaining and refusing to finish my unit tests due to the following errors;

```
Could not find a declaration file for module 'slimdom'. '/Users/wybe/Git/slimdom-sax-parser/node_modules/slimdom/dist/slimdom.umd.js' implicitly has an 'any' type.
      Try `npm i --save-dev @types/slimdom` if it exists or add a new declaration (.d.ts) file containing `declare module 'slimdom';`

import { serializeToWellFormedString } from 'slimdom';
```

and

```
TS2307: Cannot find module 'slimdom' or its corresponding type declarations.

import { serializeToWellFormedString } from 'slimdom';
```

It appears that it was unable to find file `dist/slimdom.d.ts` when my system imported `dist/slimdom.umd.js`. Possibly it was looking for `dist/slimdom.umd.d.ts`.

I've verified locally that this is fixed by adding the following line to slimdom's `package.json`:
```
"types": "./dist/slimdom.d.ts",
```

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
 	"name": "slimdom",
-	"version": "3.0.1",
+	"version": "3.0.0",
 	"description": "Fast, tiny, standards-compliant XML DOM implementation for node and the browser",
 	"author": "Stef Busking",
 	"license": "MIT",
@@ -10,8 +10,16 @@
 		"XMLSerializer",
 		"w3c"
 	],
-	"main": "dist/slimdom.umd.js",
-	"module": "dist/slimdom.esm.js",
+	"main": "./dist/slimdom.umd.js",
+	"module": "./dist/slimdom.esm.js",
+	"exports": {
+		".": {
+			"require": "./dist/slimdom.umd.js",
+			"import": "./dist/slimdom.esm.js",
+			"default": "./dist/slimdom.esm.js"
+		}
+	},
+	"types": "./dist/slimdom.d.ts",
 	"scripts": {
 		"build:clean": "rimraf dist && rimraf lib && rimraf temp",
 		"build:bundle": "tsc -P tsconfig.build.json && rollup -c",
@@ -26,7 +34,6 @@
 	"files": [
 		"dist"
 	],
-	"types": "./dist/slimdom.d.ts",
 	"repository": {
 		"type": "git",
 		"url": "git://github.com/bwrrp/slimdom.js.git"

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
 	"name": "slimdom",
-	"version": "3.0.0",
+	"version": "3.0.1",
 	"description": "Fast, tiny, standards-compliant XML DOM implementation for node and the browser",
 	"author": "Stef Busking",
 	"license": "MIT",
@@ -26,6 +26,7 @@
 	"files": [
 		"dist"
 	],
+	"types": "./dist/slimdom.d.ts",
 	"repository": {
 		"type": "git",
 		"url": "git://github.com/bwrrp/slimdom.js.git"

